packages/modeling/src/geometries/slice/calculatePlane.test.js

```diff
@@ -12,7 +12,7 @@ test('slice: calculatePlane() returns correct plans for various slices', (t) =>
   // const slice1 = create()
   // const plane1 = calculatePlane(slice1)
 
-  const slice2 = fromPoints([[0, 0], [1, 0], [1, 1]])
+  const slice2 = fromVertices([[0, 0], [1, 0], [1, 1]])
```

Comment:
Why does this say vertices, but the points are 2D?

I guess the answer is that `slice.fromPoints` can take either `vec2` or `vec3` as inputs. So we don't _need_ to rename it technically. But it is meant to be a 3D slice, so I'm good with renaming to `fromVertices` to make it more clear to users of the api.

Reply:
I came to the same conclusion. fromVertices() is probably the better name, as mixing "points' and 'vertices' in the API would be more confusing.